client/src/pages/LessonHome/LessonHome.js

```diff
@@ -70,18 +71,20 @@ const LessonHome = ({ navigation }) => {
   useEffect(() => {
     const getData = async () => {
       if (lessonData?.vocab) {
-        // only attempt to display selected vocab items
         const selectedData = lessonData.vocab.filter((item) => item.selected)
-        const formattedVocabData = selectedData.map((item) => {
+        let formattedVocabData = selectedData.map(async (item) => {
+          const imageUri = await AsyncStorage.getItem(`${item._id}/image`)
+          const audioUri = await AsyncStorage.getItem(`${item._id}/audio`)
+
           const formattedItem = {
             _id: item._id,
             name: item.original,
             body: item.translation,
-            audioURI: '',
+            audioURI: audioUri == null ? item.audioURI : audioUri,
```

Comment:
Use [=== instead of == ](https://www.scaler.com/topics/javascript/difference-between-double-equals-and-triple-equals-in-javascript/)